Makefile

```diff
@@ -84,24 +84,24 @@ devel-install: $(WEBPACK_TEST)
 # when building a distribution tarball, call webpack with a 'production' environment
 # ship a stub node_modules/ so that `make` works without re-running `npm install`
 dist-gzip: NODE_ENV=production
-dist-gzip: all cockpit-$(PACKAGE_NAME).spec
+dist-gzip: all $(PACKAGE_NAME).spec
 	mv node_modules node_modules.release
 	mkdir -p $(NODE_MODULES_TEST)
 	touch -r package.json $(NODE_MODULES_TEST)
 	touch dist/*
-	tar czf cockpit-$(PACKAGE_NAME)-$(VERSION).tar.gz --transform 's,^,cockpit-$(PACKAGE_NAME)/,' \
-		--exclude cockpit-$(PACKAGE_NAME).spec.in \
-		$$(git ls-files) cockpit-$(PACKAGE_NAME).spec dist/ node_modules
+	tar czf $(PACKAGE_NAME)-$(VERSION).tar.gz --transform 's,^,$(PACKAGE_NAME)/,' \
```

Comment:
TODO: 1) We need to omit the minor version to be added into the tar source
2) the source directory should contain <package-name>-<major-version>

Reply:
source directory should contain the major version also.

Reply:
We can take this patch. This does a lot of fixes.
We can enhance later on.